format/src/main/scala/org/scalafmt/internal/TokenOps.scala

```diff
@@ -86,7 +87,7 @@ object TreeOps {
       true
 
     case (g.Term.PrefixExpr("-"), g.Term.PrefixArg(Term.Select(tree, _), _))
```

Comment:
Should we not deconstruct Term.Select here now that it's handled in startsWithNumericLiteral?

Reply:
I want it to fall in the other case if it's not a Select.